server/liveblog/blogs/tasks.py

```diff
@@ -36,37 +37,92 @@ def publish_embed(blog_id, api_host=None, theme=None):
     return superdesk.upload.url_for_media(file_id)
 
 
-def delete_embed(blog_id):
+def delete_embed(blog_id, theme=None, output=None):
```

Comment:
Again, doesn't seem to be much of an advantage to handle different cases all in one method.